package.json

```diff
@@ -20,12 +20,19 @@
     "apollo-server-express": "^2.3.1",
     "dotenv": "^6.2.0",
     "express": "^4.16.4",
+    "express-session": "^1.17.0",
     "graphql": "^14.1.1",
     "http": "0.0.0",
+    "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.15",
+    "method-override": "^3.0.0",
+    "passport": "^0.4.1",
+    "passport-jwt": "^4.0.0",
+    "passport-local": "^1.0.0",
     "pg": "^7.17.1",
     "sequelize": "^5.21.3",
-    "sequelize-cli": "^5.5.1"
+    "sequelize-cli": "^5.5.1",
+    "uuid": "^3.4.0"
```

Comment:
:thinking: are you really using all these new packages? I'm dubious at least about `express-session`, `method-override`, `passport-local`.

Reply:
I am not using all of those new packages. I did not know how passport worked and so I ended installing and trying a lot of things that used packages that are not required anymore. How should I eliminate them from the project? Do I remove them from package.json and that's it?